scripts/upgrade

```diff
@@ -111,11 +125,14 @@ ynh_system_user_create --username=$app --home_dir=$final_path
 #=================================================
 ynh_script_progression --message="Installing Node-RED..." --weight=30
 
-chown -R $app: $final_path
+chown -R root:root $final_path
```

Comment:
Uuuuh I get very suspicious when I see this ... If root is the only owner, then that means any other user that needs to enter $final_path goes through the "others" permission (which also means any other user on the system can enter) - *or* the app runs as root

But maybe it doesn't matter for this app or there's a good reason for this :s

Reply:
I have actually no good explanation of why I put this line... Let's revert it.